### PR TITLE
SISRP-12707 - UX DESIGN: Personalize Profile Avatar for Staff and Faculty

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-12707

### CODE REVIEW

File: src/assets/templates/widgets/gear_popover.html [hunk 1]
```diff
@@ -4,9 +4,11 @@
   data-ng-class="{true:'cc-header-icon-selected'}[api.popover.status('cc-popover-menu')]"
   title="Settings">
     <div class="cc-flex cc-flex-align-center">
-      <span data-ng-if="api.user.profile.hasPhoto" class="cc-popover-gear-round-profile cc-popover-gear-round-profile-has-photo">
+      <span data-ng-if="!api.user.profile.roles.student && !api.user.profile.hasPhoto"
```
Comment (christianvuerings) [inline src/assets/templates/widgets/gear_popover.html]: @dfkaye why are we adding the `api.user.profile.roles.student` logic in here? Shouldn't that stay the same?

E.g. what happens when a delegate is seeing this?

Comment (dfkaye) [inline src/assets/templates/widgets/gear_popover.html]: @christianvuerings My understanding is:
+ Students viewing themselves see either their own photo or the gray profile image.
+ Non-students viewing themselves see either their own photo or the *bear* profile image.
+ A delegate sees *only* the gray profile image for the student.

Comment (christianvuerings) [inline src/assets/templates/widgets/gear_popover.html]: @dfkaye in this PR (just confirmed with Flint) we should replace the gray profile image (in the top navigation) for every case.

**Comment by dfkaye** [comment]
@christianvuerings second pass ready for review

File: src/assets/templates/widgets/gear_popover.html [hunk 2]
```diff
@@ -4,11 +4,8 @@
   data-ng-class="{true:'cc-header-icon-selected'}[api.popover.status('cc-popover-menu')]"
   title="Settings">
     <div class="cc-flex cc-flex-align-center">
-      <span data-ng-if="api.user.profile.hasPhoto" class="cc-popover-gear-round-profile cc-popover-gear-round-profile-has-photo">
-      </span>
-      <span data-ng-if="!api.user.profile.hasPhoto" class="cc-popover-gear-round-profile cc-popover-gear-round-profile-no-photo">
```
Comment (christianvuerings) [inline src/assets/templates/widgets/gear_popover.html]: @dfkaye are we still using `cc-popover-gear-round-profile-no-photo` anywhere? If not, can we remove it from the CSS?

Comment (dfkaye) [inline src/assets/templates/widgets/gear_popover.html]: @christianvuerings : looks like we're not using it anywhere ~ removing...

**Comment by christianvuerings** [comment]
@dfkaye thanks for this change - definitely don't forget to tag me on the PR when you've updated it (will get you faster merges)

**Pull request merged**